stacker/context.py

```diff
@@ -70,7 +71,11 @@ def load_config(self, conf_string):
             self.bucket_name = bucket_name
         tags = self.config.get("tags", None)
         if tags is not None:
-            self.tags = dict([(str(tag_key), str(tag_value)) for tag_key, tag_value in tags.items()])
+            self.tags = dict([(str(tag_key), str(tag_value)) for tag_key,
+                              tag_value in tags.items()])
+        lookups = self.config.get("lookups", {})
+        for key, handler in lookups.iteritems():
```

Comment:
You only need to register the lookup in the config if it is a custom one that isn't provided by stacker. https://github.com/remind101/stacker_blueprints/pull/24/files#diff-8f84670404b5e0466057d169e260c840R58 is an example of that. Anyone defined within stacker should register itself via: https://github.com/remind101/stacker/pull/194/files#diff-00537bebd4f02f9f6ab51659e6716ef9R51. Output was the only one I did for now, but KMS etc. should be translated to stacker provided lookups that you don't have to do anything special to register in the config. Does that answer your questio?